runtime/runtime.go

```diff
@@ -308,6 +316,7 @@ func (rt *Runtime) newEVM(stateDB *statedb.StateDB, clauseIndex uint32, txCtx *x
 		BlockNumber: new(big.Int).SetUint64(uint64(rt.ctx.Number)),
 		Time:        new(big.Int).SetUint64(rt.ctx.Time),
 		Difficulty:  &big.Int{},
+		BaseFee:     &big.Int{},
```

Comment:
We used to have a var here, is it ok this way now?

Reply:
yes, this will be changed with the base-fee branch

Reply:
Clarified, thanks.